<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageInfoDialog</class>
 <widget class="QDialog" name="ImageInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>252</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>32</number>
     </property>
     <item row="1" column="1" colspan="3" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="dateLab">
       <property name="text">
        <string>无</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>存储路径：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="heightLab">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="pathLab">
       <property name="text">
        <string>无</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="widthLab">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>创建时间：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>长度：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>宽度：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="nameLab">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
       <property name="text">
        <string>无</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
